graylog2-web-interface/src/views/components/searchbar/SearchButton.tsx

```diff
@@ -53,19 +54,32 @@ type Props = {
   displaySpinner?: boolean,
 };
 
-const onButtonClick = (e: MouseEvent, disabled: Boolean) => {
+const onButtonClick = (e: MouseEvent, disabled: Boolean, triggerTelemetry: () => void) => {
   if (disabled) {
     e.preventDefault();
     QueryValidationActions.displayValidationErrors();
   }
+
+  triggerTelemetry();
```

Comment:
We can call `sendTelemetry` here directly, no need for an extra function.

Reply:
Actually, we can't because the function is defined outside of the component so we need to pass the function or move it inside the scope of the component.